kroxylicious-integration-tests/src/test/java/io/kroxylicious/proxy/filter/authorization/ClientAuthzIT.java

```diff
@@ -688,13 +695,18 @@ public void run(ClientFactory clientFactory) throws InterruptedException {
 
                     try (var consumer2 = clientFactory.newConsumer(consumerUser, INTEGER_DESERIALIZER, Map.of(
                             ConsumerConfig.CLIENT_ID_CONFIG, "consumer2",
-                            ConsumerConfig.GROUP_ID_CONFIG, GROUP_1,
-                            ConsumerConfig.GROUP_INSTANCE_ID_CONFIG, GROUP_1 + "-instance-1",
+                            ConsumerConfig.GROUP_ID_CONFIG, GROUP_2,
```

Comment:
I figured this second consumer was easier to reason about by giving it a new group, so you know that the earlier consumer hasn't impacted it